### PR TITLE
Update readme to reflect how to override the created docker tag

### PR DESCRIPTION
**Pull request by brujoand**
The readme suggested that setting ´tag=yourtag´ would override the docker image tag for the created image. But this did not work, and this PR should fix that.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -51,8 +51,10 @@ upgrade-build: #TODO: reenable it
 
 .PHONY: osdk-image
 osdk-image:
+ifndef IMAGE_TAG
+ override IMAGE_TAG = quay.io/rhpipeline/openshift-pipelines-operator:test
```
Comment (chmouel) [inline Makefile]: I think you can just use : 

```
IMAGE_TAG := quay.io/rhpipeline/openshift-pipelines-operator:test
```

and having `make IMAGE_TAG=foo` would redefine it,

Comment (brujoand) [reply Makefile]: ah, you are absolutely right. 👍

**Comment by brujoand** [comment]
I've reverted the change to the makefile, I guess the update to the readme might still me useful.

**Comment by brujoand** [comment]
/retest 
go dependency management is not my strong suit, but this seem to do the trick.

**Comment by brujoand** [comment]
Awesome, I've reverted the change to ´go.mod˙ now, assuming we can just squash these commits. As for the readme, I think I've found some other things that needs to be updated but I need some help deciphering what changes are required. I've reported this as #94 if anyone as time to take a look.

**Pull request merged**